ghost/i18n/locales/ru/portal.json

```diff
@@ -1,173 +1,173 @@
+    "There was an error processing your payment. Please try again.": "Произошла ошибка при обработке вашего платежа. Попробуйте ещё раз.",
+    "This site is invite-only, contact the owner for access.": "Доступ к материалам этого сайта возможен только по приглашению. Для получения доступа свяжитесь с владельцем сайта.",
+    "This site is not accepting payments at the moment.": "В данный момент сайт не принимает платежи.",
+    "To complete signup, click the confirmation link in your inbox. If it doesn't arrive within 3 minutes, check your spam folder!": "Чтобы завершить регистрацию, нажмите на ссылку подтверждения в электронном письме, которое мы прислали. Если письмо не пришло в течение 3 минут, проверьте папку «Спам»!",
+    "To continue to stay up to date, subscribe to {{publication}} below.": "Чтобы оставаться в курсе событий, подпишитесь на {{publication}} ниже.",
+    "Try free for {{amount}} days, then {{originalPrice}}.": "Попробуйте бесплатно в течение {{amount}} дня(ей), затем за {{original Price}}.",
+    "Unlock access to all newsletters by becoming a paid subscriber.": "Получите доступ ко всем рассылкам, оформив платную подписку.",
+    "Unsubscribe from all emails": "Отписаться от всех рассылок",
+    "Unsubscribed": "Отписан",
+    "Unsubscribed from all emails.": "Отписан от всех рассылок.",
+    "Unsubscribing from emails will not cancel your paid subscription to {{title}}": "Отписка от рассылки не отменит вашу платную подписку на {{title}}",
+    "Update": "Обновить",
     "Update your preferences": "Обновить настройки",
-    "Verification link sent, check your inbox": "",
-    "Verify your email address is correct": "",
-    "View plans": "",
-    "We couldn't unsubscribe you as the email address was not found. Please contact the site owner.": "Мы не смогли отменить вашу подписку, так как адрес электронной почты не найден. Пожалуйста, свяжитесь с владельцем сайта.",
-    "Welcome back, {{name}}!": "",
-    "Welcome back!": "",
-    "Welcome to {{siteTitle}}": "",
-    "When an inbox fails to accept an email it is commonly called a bounce. In many cases, this can be temporary. However, in some cases, a bounced email can be returned as a permanent failure when an email address is invalid or non-existent.": "",
-    "Why has my email been disabled?": "",
+    "Verification link sent, check your inbox": "Ссылка для подтверждения отправлена, проверьте свой почтовый ящик",
+    "Verify your email address is correct": "Проверьте правильность вашего email адреса.",
+    "View plans": "Ознакомиться с планами",
+    "We couldn't unsubscribe you as the email address was not found. Please contact the site owner.": "Мы не смогли отписать вас, так как указанный email адрес не был найден. Пожалуйста, свяжитесь с владельцем сайта.",
+    "Welcome back, {{name}}!": "Добро пожаловать, {{name}}, с возвращением!",
```

Comment:
Would this be more appropriate?

```
Добро пожаловать обратно, {{name}}!
```

Reply:
Good alternative, your version also correct.